Toolset/palettes/tutorial/courses/First Run/tutorials/BMI Calculator/lessons/App Lesson.txt

```diff
@@ -139,7 +143,7 @@ action
 end step
 
 step "Set Calculator Header Style"
-	Set the action style of the header widget to "names"
+	Select "names" from the 'Action Display Style' drop down menu.
```

Comment:
Since 'names' is text that appears in the user interface, maybe it should be in single quotes?  My assumption is that you've been using "..." for text the user types in, and '...' for text the user sees.